build/base-web/Dockerfile

```diff
@@ -12,11 +12,8 @@ ENV DJANGO_PORT 80
 ENV NODEJS_PORT 3000
 ENV DEBUG       false
 
-# Install re2 for Python
-
 RUN pip3 install cython
-RUN git clone https://code.googlesource.com/re2 && cd re2 && make && make install && cd / && pip3 install https://github.com/andreasvc/pyre2/archive/3e01eba6ba3eabd1359ef5e16c938c8866deea70.zip
-
+RUN apt-get update -y && apt-get install -y npm varnish && apt-get clean
+RUN pip3 install google-re2
```

Comment:
Didnt we say we'd move this into requirements?

Reply:
Happy to move it